bin/index.js

```diff
@@ -104,50 +101,25 @@ const askQuestions2 = () => {
       type: 'list',
       name: '_actionType',
       message: 'What action is the certificate asserting?',
-      choices: ["REGISTER", "UPDATE"],
+      choices: ["REGISTER", "DE_REGISTER"],
       default: "REGISTER"
     },
     {
-      name: "_releaseNumber",
+      name: "_releaseComment",
```

Comment:
Comment might be present also when de registering and in that case it is not a release.
```suggestion
      name: "comment",
```

Reply:
Fixed on d41e940